client/i18n/locales/english/translations.json

```
@@ -719,7 +719,7 @@
   },
   "signout": {
     "heading": "Sign out of your account",
-    "p1": "Warning: Signing out will stop saving your progress.",
+    "p1": "Warning: Signing out will stop your progress being saved.",
```

```suggestion
    "p1": "Warning: If you continue, it will stop saving your progress to your account.",
```

I really want to use an active voice. Plus you are right, we do not need to mention "Signing out" as a verb.

---

```suggestion
    "p1": "Warning: If you continue, it will stop saving your progress.",
```

Another shorter alternative but still active voice. I'll let you choose.

---

The issue is that neither of the alternatives makes grammatical sense and we're pushed into writing them by avoiding the passive voice.

We *could* say "If you continue, we will stop saving your progress.", which is fine grammatically, but (ironically) comes across as passive aggressive.

So, if we avoid repeating 'signing out' we could go with:
```suggestion
    "p1": "Warning: If you continue, your progress will no longer be saved.",
```

---

That sound exactly like what I wanted.